torch/_inductor/lowering.py

```diff
@@ -4156,11 +4158,35 @@ def fn(idx):
     return rv
 
 
-fallback_avg_pool2d = fallback_handler(
-    aten.avg_pool2d.default, add_to_fallback_set=False
+fallbacks_avg_poolnd = (
+    fallback_handler(aten.avg_pool1d.default, add_to_fallback_set=False),
+    fallback_handler(aten.avg_pool2d.default, add_to_fallback_set=False),
+    fallback_handler(aten.avg_pool3d.default, add_to_fallback_set=False),
 )
 
 
+@register_lowering(aten.avg_pool1d, type_promotion_kind=None)
+def avg_pool1d(
```

Comment:
Looks like `avg_pool1d` is `CompositeImplicitAutograd` and dispatches to `avg_pool2d`
https://github.com/pytorch/pytorch/blob/f1a393c029e71ea87f3d08f783a31b45ccd3596e/aten/src/ATen/native/Pooling.cpp#L131

This means it shouldn't ever appear in inductor's fx graph, so it doesn't need a lowering. We do still need `avg_pool3d` though.